richreview/public/webapps/richreview/r2/app.js

```diff
@@ -142,7 +142,7 @@
                     r2App.cur_recording_pieceaudios[r2App.cur_recording_pieceaudios.length-1].DrawPieceDynamic(null, r2.annot_canv_ctx, true); // force
                 }
             }
-            r2.spotlightCtrl.drawDynamicSceneTraces(r2.annot_canv_ctx);
+            //r2.spotlightCtrl.drawDynamicSceneTraces(r2.annot_canv_ctx);
```

Comment:
Please excise the unused code line, rather than leaving it commented out. Keeping the code base comment-less is important for better readability.